kamon-stackdriver/src/main/scala/kamon/stackdriver/StackdriverMetricReporter.scala

```diff
@@ -115,23 +122,38 @@ class StackdriverMetricReporter extends MetricReporter {
       .build()
   }
 
-  def histogram(v: MetricDistribution, timeInterval: TimeInterval): TimeSeries = {
-    val distribution = histogramToDistributionConverter.histogramToDistribution(v.distribution.buckets, v.distribution.count)
+  def histogram(v: Distributions, timeInterval: TimeInterval): Seq[TimeSeries] = {
+    v.instruments.map{ i =>
```

Comment:
I wasn't sure how I should play with `instruments`. I cannot find any documentation about this. I modeled myself on other reporters.